build/gen_stub.php

```diff
@@ -3035,11 +3129,59 @@ public function getDeclaration(array $allConstInfos): string {
         );
         $code .= implode("", $flagsCode);
 
-        $code .= "\tzend_string_release(property_{$propertyName}_name);\n";
+        $code .= $stringRelease;
 
         return $code;
     }
 
+    /**
+     * Get an array of three strings:
+     *   - declaration of zend_string, if needed, or empty otherwise
+     *   - usage of that zend_string, or usage with ZSTR_KNOWN()
+     *   - freeing the zend_string, if needed
+     *
+     * @param string $propName
+     * @return string[]
+     */
+    private function getString( string $propName ): array {
+        // Generally strings will not be known
+        $nameCode = "property_{$propName}_name";
+        $result = [
+            "\tzend_string *$nameCode = zend_string_init(\"$propName\", sizeof(\"$propName\") - 1, 1);\n",
+            $nameCode,
+            "\tzend_string_release($nameCode);\n"
+        ];
+        $minPhp = $phpVersionIdMinimumCompatibility ?? PHP_84_VERSION_ID;
```

Comment:
The fallback value (`PHP_84_VERSION_ID`) will also have to be updated every time a new PHP release adds a new known string... I would prefer if we could prevent this extra step, because this reference is difficult to spot.

Reply:
sure - `end( ALL_PHP_VERSION_IDS )` also works